apps/site/navigation.json

```diff
@@ -139,10 +139,6 @@
               "link": "/learn/getting-started/introduction-to-nodejs",
               "label": "components.navigation.learn.gettingStarted.links.introductionToNodejs"
             },
-            "howToInstallNodejs": {
-              "link": "/learn/getting-started/how-to-install-nodejs",
-              "label": "components.navigation.learn.gettingStarted.links.howToInstallNodejs"
```

Comment:
You have to also remove the i18n key. It's on packages/i18n

Reply:
removed the keys in the latest commit sorry for not noticing that.